Sources/CreateAPI/Generator/Generator+Schemas.swift

```diff
@@ -99,7 +99,8 @@ extension Generator {
             return options.entities.include.contains(name)
         }
         if !options.entities.exclude.isEmpty {
-            return !options.entities.exclude.contains(name)
+//            return !options.entities.exclude.contains(name)
+            return true
```

Comment:
Debugging?

Reply:
Oops

Reply:
Your review seems to be attached to a certain commit that I did while working on this. This is not in the final version.